PrBuild/config.json

```diff
@@ -1,12 +1,14 @@
-{      
-    "repo_owner": "pniko",
-    "repo_name": "detox-appcenter",
-
-    "appcenter_owner_type": "users",
-    "appcenter_owner": "pniko",
-    "appcenter_app": "detox-appcenter",
-
-    "branch_template": "master",
-    "branch_regex": "(skip|no-build)"
+{
+    "repos": [
+        {
+            "repo_owner": "pniko",
```

Comment:
Let's add a property here for "provider" : "github" in preparation for the future. We're going to add VSTS support in the near future and it makes sense to put this all in a single webhook handler.

Reply:
Added, thanks.